app/models/task.py

```diff
@@ -1,5 +1,43 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True,autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, default=None) #time_date, need to have option for null or None
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+
+    def complete_or_not(self):
+        '''
+        Converts 'completed_at' task attribute to Boolean for 'is_completed' variable in response body
+        '''
+        if self.completed_at is not None:
+            return True
+        else:
+            return False
```

Comment:
This can be simplified to `return self.completed_at is not None` or even to:

```suggestion
        return bool(self.compleated_at)
```

Which would let you get rid of this method.

It was a good instinct to pull this out for readability though!  Just remember that Python has a _lot_ of useful stuff built in.  :smile: